eos/utils/integrate_TEST.cc

```diff
@@ -73,18 +73,29 @@ class IntegrateTest :
             std::cout << "\\int_0.0^10.0 f3(x) dx = " << q3 << ", eps = " << std::abs(i3 - q3) / q3 << " over 16 points" << std::endl;
             TEST_CHECK_RELATIVE_ERROR(i3, q3, eps);
 
-            double q4 = integrate1D(std::function<double (const double &)>(&f4), 16, 1.0, std::exp(1)), i4 = 1.0;
+            auto f4obj = std::function<double (const double &)>(&f4);
+            double q4 = integrate1D(f4obj, 16, 1.0, std::exp(1)), i4 = 1.0;
```

Comment:
Can you separate this and the following two cleanups from the cubature part, i.e., in a separate commit?